src/dmd/declaration.h

```diff
@@ -442,14 +442,45 @@ enum ILS
 
 enum BUILTIN
```

Comment:
` : ubyte`

Reply:
Thats a header file and we need C++98 compatibility.